schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties": {
+              "abstract": {
+                "type": "string",
+                "minLength": 1
+              },
+              "language": {
+                "const": "en"
+              }
+            },
+            "additionalProperties": false
+          }
+        ],
+        "additionalItems": {
+          "type": "object",
+          "properties": {
+            "abstract": {
+              "type": "string",
+              "minLength": 1
+            },
+            "language": {
+               "allOf": [
+                 {"$ref": "#/definitions/language"},
+                 {"not": {"enum": ["en"]}}
+             ]
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "affiliation": {
+        "title": "Affiliation",
+        "description": "Institutional affiliation.",
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The complete name of the affiliation.",
+            "type": "string",
+            "minLength": 1
+          },
+          "ror_id": {
+            "title": "ROR ID",
+            "description": "Research Organization Registry ID.",
+            "type": "string",
+            "pattern": "^https://ror\\.org/0\\w{6}\\d{2}$",
+            "examples": [
+              "https://ror.org/05bnh6r87",
+              "https://ror.org/05gzmn429",
+              "https://ror.org/01ggx4157",
+              "https://ror.org/03c3r2d17"
+            ]
+          }
+        },
+        "required": [
+          "name"
+        ],
+        "additionalProperties": false
+      },
+      "category": {
+        "title": "arXiv category",
+        "description": "This is the abbreviated form of the arXiv category using the category identifier",
+        "type": "string",
+        "pattern": "^([a-z]{3,}-[a-z]{2,}|[a-z]{2,}-?[a-z]+\\.[A-Z]{2})$",
+        "$comment": "TODO: enumeration"
+      },
+      "collaboration": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The fullest representation of the collaborations's name",
+            "type": "string",
+            "minLength": 1
+          }
+        }
+      },
+      "comment": {
+        "title": "Comment",
+        "type": "string",
+        "minLength": 1
+      },
+      "conference_info": {
+        "title": "Conference information",
+        "type": "object",
+        "$comment": "based on subset of INSPIRE's definition",
+        "properties": {
+          "conference_titles": {
+            "title": "List of conference titles",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "minLength": 1
+            },
+            "minItems": 1,
+            "uniqueItems": true
+          },
+          "cnum": {
```

Comment:
this is an internal INSPIRE identifier. I'm not sure you want to use those, as they're HEP specific. There's [some work happening](https://www.project-freya.eu/en/blogs/blogs/towards-persistent-identification-of-conferences) on conference PIDs, but AFAIK there's nothing concrete yet.

Reply:
Thanks, I thought I had seen `cnum` used in other contexts, but I confirm with a quick search on arXiv that they're only being used in a handful of HEP papers (via the legacy `comments` field).